untrusted/test_enclave/src/config.xml

```diff
@@ -0,0 +1,15 @@
+<EnclaveConfiguration>
+    <ProdID>0</ProdID>
+    <ISVSVN>0</ISVSVN>
+    <TCSMaxNum>1</TCSMaxNum>
+    <TCSNum>1</TCSNum>
+    <TCSMinPool>1</TCSMinPool>
+    <TCSPolicy>1</TCSPolicy>
+    <StackMaxSize>0x40000</StackMaxSize>
+    <HeapMaxSize>0x100000</HeapMaxSize>
+    <HeapInitSize>0x100000</HeapInitSize>
+    <HeapMinSize>0x100000</HeapMinSize>
+    <DisableDebug>0</DisableDebug>
+    <MiscSelect>0</MiscSelect>
+    <MiscMask>0xFFFFFFFF</MiscMask>
+</EnclaveConfiguration>
```

Comment:
As noted, there is code that handles constructing this file in a repeatable way in the repo, we should use that code.

Reply:
let me know if https://github.com/mobilecoinfoundation/sgx/issues/4 captures this desire, or if you think these changes are taking on unjustifiable debt and should be avoided from the get go.